docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -964,19 +1049,21 @@ particularly interesting when using special options to control image pulls.
 
 Valid options for `[Image]` are listed below:
 
-| **[Image] options**                 | **podman image pull equivalent**                |
-|-------------------------------------|-------------------------------------------------|
-| AllTags=true                        | --all-tags                                      |
-| Arch=aarch64                        | --arch=aarch64                                  |
-| AuthFile=/etc/registry/auth\.json   | --authfile=/etc/registry/auth\.json             |
-| CertDir=/etc/registery/certs        | --cert-dir=/etc/registery/certs                 |
-| Creds=myname\:mypassword            | --creds=myname\:mypassword                      |
-| DecryptionKey=/etc/registery\.key   | --decryption-key=/etc/registery\.key            |
-| Image=quay.io/centos/centos\:latest | podman image pull quay.io/centos/centos\:latest |
-| OS=windows                          | --os=windows                                    |
-| PodmanArgs=--os=linux               | --os=linux                                      |
-| TLSVerify=false                     | --tls-verify=false                              |
-| Variant=arm/v7                      | --variant=arm/v7                                |
+| **[Image] options**                    | **podman image pull equivalent**                |
+|----------------------------------------|-------------------------------------------------|
+| AllTags=true                           | --all-tags                                      |
+| Arch=aarch64                           | --arch=aarch64                                  |
+| AuthFile=/etc/registry/auth\.json      | --authfile=/etc/registry/auth\.json             |
+| CertDir=/etc/registery/certs           | --cert-dir=/etc/registery/certs                 |
+| ContainersConfModule=/etc/nvidia\.conf | --module=/etc/1.conf                            |
```

Comment:
There's still a `1.conf` on the right

Reply:
Thanks for catching it.
In addition, it seems that the line issue also had something to do with the length of the line. So, I has to both escape the `.` and remove some letters. Not sure where this is coming from though